scripts/NER_block.py

```
@@ -0,0 +1,199 @@
+    # Поиск даты
+    date, text = text_to_date(text)
+    # Поиск времени
+    time, text = text_to_time(text)
+
+    if date:
+        int_date = int(date.split(".")[0]) + (30 * int(date.split(".")[1])) # Дата в виде целого числа
+        today_date = datetime.now().day + datetime.now().month * 30
+        if today_date > int_date:
+            year = datetime.now().year + 1 # Если дата уже прошла, то добавляем год
+        else:
+            year = datetime.now().year
```

Ruff (0.7.0)

Use ternary operator `year = datetime.now().year + 1 if today_date > int_date else datetime.now().year` instead of `if`-`else`-block

(SIM108)